README.md

```diff
@@ -0,0 +1,305 @@
+
+-   [Requirements](#Requirements)
+    -   [Linux-Debian-Ubuntu-Mint](#Linux-Debian-Ubuntu-Mint)
+    -   [Linux Fedora-CentOS-RHEL](#Linux-Fedora-CentOS-RHEL)
+    -   [Mac OS X](#Mac-OS-X)
+    -   [Windows](#Windows)
+
+-   [Installation](#Installation)
+    -   [From PyPi](#From-PyPi)
+    -   [From sources](#From-sources)
+
+-   [Additional packages](#Additional-packages)
+
+-   [Testing](#Testing)
+
+-   [Pyccel Container Images](#Pyccel-Container-Images)
+
+## User Documentation
+
+-   [Quick-start Guide](./tutorial/quickstart.md)
+
+-   [C/Fortran Compilers](./tutorial/compiler.md)
+
+-   [Decorators](./tutorial/decorators.md)
+
+-   [Header Files](./tutorial/header-files.md)
+
+-   [Templates](./tutorial/templates.md)
+
+-   [N-dimensional Arrays](./tutorial/ndarrays.md)
+
+-   Supported libraries/APIs
+    -   [OpenMP](./tutorial/openmp.md)
+    -   [Numpy](./tutorial/numpy-functions.md)
+
+## Pyccel Installation Methods
+
+Pyccel can be installed on virtually any machine that provides Python 3, the pip package manager, a C/Fortran compiler, and an Internet connection.
+Some advanced features of Pyccel require additional non-Python libraries to be installed, for which we provide detailed instructions below.
+
+Alternatively, Pyccel can be deployed through a **Linux Docker image** that contains all dependencies, and which can be setup with any version of Pyccel.
+For more information, please read the section on [Pyccel container images](#Pyccel-Container-Images).
+
+## Requirements
+
+First of all, Pyccel requires a working Fortran/C compiler
+
+For Fortran it supports
+
+-   [GFortran](https://gcc.gnu.org/fortran/)
+-   [Intel® Fortran Compiler](https://software.intel.com/en-us/fortran-compilers)
+-   [PGI Fortran](https://www.pgroup.com/index.htm)
+
+For C it supports
+
+-   [GCC](https://gcc.gnu.org/)
+-   [Intel® Compiler](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/dpc-compiler.html)
+-   [PGI](https://www.pgroup.com/index.htm)
+
+In order to perform fast linear algebra calculations, Pyccel uses the following libraries:
+
+-   [BLAS](http://www.netlib.org/blas/)(Basic Linear Algebra Subprograms)
+-   [LAPACK](http://www.netlib.org/lapack/)(Linear Algebra PACKage)
+
+Finally, Pyccel supports distributed-memory parallel programming through the Message Passing Interface (MPI) standard; hence it requires an MPI library like
+
+-   [Open-MPI](https://www.open-mpi.org/)
+-   [MPICH](https://www.mpich.org/)
+-   [Intel® MPI Library](https://software.intel.com/en-us/mpi-library)
+
+We recommend using GFortran/GCC and Open-MPI.
+
+Pyccel also depends on several Python3 packages, which are automatically downloaded by pip, the Python Package Installer, during the installation process. In addition to these, unit tests require the _scipy_, _mpi4py_, _pytest_ and _coverage_ packages, while building the documentation requires [Sphinx](http://www.sphinx-doc.org/).
```

Comment:
I've just noticed that we don't mention OpenMP in the requirements. Could you add it please